src/main/java/com/sap/sgs/phosphor/fosstars/model/Confidence.java

```diff
@@ -85,7 +85,7 @@ static double make(Value... values) {
       return MAX;
     }
 
-    return (MAX * unknown) / values.length;
+    return MAX * (values.length - unknown) / values.length;
```

Comment:
Is there a test case to check
when values.length = 0?

Test case for this function make()